modules/Core/classes/Events/AnnouncementCreatedEvent.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+class AnnouncementCreatedEvent extends AbstractEvent implements DiscordDispatchable {
+
+    public User $creator;
+    public string $header;
+    public string $message;
+
+    public function __construct(User $creator, string $header, string $message) {
+        $this->creator = $creator;
```

Comment:
Uncaught TypeError
AnnouncementCreatedEvent::__construct(): Argument #1 ($creator) must be of type User, string given, called in /var/www/dev/core/classes/Misc/Announcements.php on line 187